src/jsrm/math_utils.py

```diff
@@ -40,3 +42,33 @@ def assign_block_diagonal(i, _b):
     )
 
     return b
+
+@jit
+def blk_concat(
+    a: Array
+) -> Array:
+    """
+    Concatenate horizontally (along the columns) a list of N matrices of size (a, b) to create a single matrix of size (a, b * N).
+
+    Args:
+        a (Array): matrices to be concatenated of shape (N, a, b)
+
+    Returns:
+        Array: concatenated matrix of shape (a, N * b)
```

Comment:
Should be `b (Array): concatenated matrix of shape (a, N * b)`. Also, there is a namespace conflict as you call both the shape (a, b) and the variables (a, b)